sig-scheduling/README.md

```diff
@@ -80,6 +80,9 @@ The following [subprojects][subproject-definition] are owned by sig-scheduling:
 ### kueue
 - **Owners:**
   - [kubernetes-sigs/kueue](https://github.com/kubernetes-sigs/kueue/blob/main/OWNERS)
+### kwok
```

Comment:
Thanks @nikhita for helping create the repo!

One qq: is it possible for this project to be listed at 2 sigs - basically, the consensus was it'll be **co-sponsored** by sig-scalability and sig-scheduling? Do you think it makes sense and not sure we had a similar case before?

Reply:
> is it possible for this project to be listed at 2 sigs

We currently don't have automation to support this.

Today, you can add a description to the repo to mention that the repo involves heavy collaboration with sig-scalability.

> Do you think it makes sense

IMHO having a single owning SIG is easier operationally because there would be a single SIG responsible and accountable for the repo. 

Having said that, deciding whether we should support multiple SIGs in steering committee territory so:
cc @kubernetes/steering-committee

Reply:
My personal thoughts - Multiple SIGs can participate in a subproject, there are quite a few that do currently - but when it comes to taking ownership, reporting on its status and be the ones that are ultimately responsible for taking on that project, it needs to goto a single owner.

Reply:
agreed, nothing prevents including folks from multiple sigs in the OWNERS file, but a single primary sig seems good

Reply:
Thanks all for the input. I will have it primarily sponsored by sig-scheduling.